packages/frontend/src/components/dialogs/ProxyConfiguration/styles.module.scss

```diff
@@ -34,6 +30,9 @@
 
 .proxyLabel {
   margin-bottom: 6px;
+  max-width: 320px;
```

Comment:
This is a wrong approach. The buttons get shifted when this element reaches 320px in width:

<img width="470" height="521" alt="Image" src="https://github.com/user-attachments/assets/b2483b9c-a807-41c9-ab79-2bea1d587c90" />

Reply:
I can only reproduce that when I narrow the window to less than 500px which is an extreme edge case. And the form is still usable , only doesn't looks nice. I consider that good enough.

But if you have know one, a better solution would be welcome

Reply:
> I can only reproduce that when I narrow the window to less than 500px

Not the case for me. try playing around with zoom maybe.

> I consider that good enough.

I agree.